### ISSUE TITLE
Inaccurate values sometimes returned through the Packet object in hand recognition 

### ISSUE THREAD

**Issue by harsha-main** [google/mediapipe#794]
I've downloaded and run the sample Android project that was shared. I made some modifications to it, by displaying some Views near the joints of the fingers. 

I have noticed that the value of landmark.getX() and landmark.getY() methods aren't in line with the annotation's red dots of the joints, especially when the hand is abruptly moved to some distance
The way I'm getting the coordinates is by the following method 
`float x =  landmark.getX() * screenWidth; `
and similarly for the y coordinates for each joint.

I'm loading a rectangular view right at the x,y obtained by the above formula, hoping the top right corner of the view will be placed exactly at the red dot given in the overlay.

While this approach sometimes works as expected, when you move your hand to say right side, my view _travels lesser distance compared to the annotation_, similarly for the other sides. 

To sum it up, I hope to get the values given by the landmark object will be exactly where the detected red dots are.

**Comment by fanzhanggoogle** [comment]
I wonder which device are you using? It seems that there could be a difference between the input texture size (send to the pipeline) vs the size of the rendered screen.

**Comment by harsha-main** [comment]
@fanzhanggoogle 
I have tried this on a budget Redmi 7A phone, and the Samsung S8+, which has a better processor.

**Comment by sgowroji** [comment]
Hi @harsha-main, Could you please test the above with the latest [release](https://github.com/google/mediapipe/tree/master/mediapipe/examples/android/src/java/com/google/mediapipe/apps/handdetectiongpu).  And please write us back if you still see the same error.
